app/templates/content/framework-digital-services.html

```diff
@@ -143,13 +139,13 @@ <h2 id="submit-services-to-the-digital-services-framework">
         You’ll be asked to describe and show evidence of your capabilities when applying for a place on the framework. As part of this process, you’ll need to demonstrate that your methodology meets the <a href="https://www.gov.uk/service-manual">Government Service Design Manual</a> and the <a href="https://www.gov.uk/service-manual/digital-by-default">Digital by Default Service Standard</a> for new or redesigned digital services.
       </p>
       <p>
-        Applications to the framework are evaluated by Government Digital Service (GDS) subject matter experts covering all 8 digital capabilities.
+        Applications to the framework are evaluated by Government Digital Service (GDS) subject matter experts covering all 7 digital capabilities.
       </p>
       <p>
         A place on the framework is no guarantee of business to a supplier. For each digital project request for proposal (RFP) suppliers are invited to respond to, they’ll go through technical, cultural and commercial evaluation.
       </p>
       <p>
-        You can find the <a rel="external" href="http://ccs-agreements.cabinetoffice.gov.uk/contracts/rm1043">Digital Services framework</a> agreement on the Crown Commercial Service (CCS) website.
+        You can find the <a rel="external" href="http://ccs-agreements.cabinetoffice.gov.uk/contracts/rm1043ii">Digital Services framework</a> agreement on the Crown Commercial Service (CCS) website.
```

Comment:
This link seems to lead to a page of search results whereas the old version went straight to a page on DSF, is this correct?